src/Scouting.elm

```diff
@@ -0,0 +1,100 @@
+module Scouting exposing (Model, Msg, init, update, view)
+
+import Browser
+import Html
+import Html.Attributes as Attributes
+import Html.Events as Events
+
+
+main : Program () Model Msg
+main =
+    Browser.sandbox
+        { init = init
+        , update = update
+        , view = view
+        }
+
+
+type Msg
+    = NewTeam String
+    | NewScouter String
+    | NewGame String
+    | Validate
+
+
+type alias Model =
+    { team : String
+    , scouter : String
+    , game : String
+    , button : Bool
+    }
+
+
+init : Model
+init =
+    { team = ""
+    , scouter = ""
+    , game = ""
+    , button = False
+    }
+
+
+update : Msg -> Model -> Model
+update msg model =
+    case msg of
+        NewTeam team ->
+            { model | team = team }
+
+        NewScouter scouter ->
+            { model | scouter = scouter }
+
+        NewGame game ->
+            { model | game = game }
+
+        Validate ->
+            { model | button = True }
+
+
+view : Model -> Html.Html Msg
+view model =
+    if model.button then
+        Html.div []
+            [ Html.div []
+                [ Html.text ("Scouter name " ++ model.scouter)
```

Comment:
You can use a pipe when you add trailing parentheses.
For instance:
```elm
[ Html.text <| "Scouter name " ++ model.scouter
```
It can usually easily improve readability and formatting.